client/src/App.tsx

```diff
@@ -1,22 +1,37 @@
-import React, { FC } from 'react';
+import React, { FC, useEffect } from 'react';
 import { BrowserRouter, Routes, Route } from 'react-router-dom';
 import Home from './Pages/Home';
 import Login from './Pages/Login';
 import MapMain from './Pages/MapMain';
 import CalendarMain from './Pages/CalendarMain';
 import CommunityEvents from './Pages/CommunityEventsMain';
 import 'bootstrap/dist/css/bootstrap.min.css';
+import './App.css';
+import Profile from './Pages/Profile';
+import axios from 'axios';
+import { setUser } from './state/features/userProfile/userProfileSlice';
+import { useAppDispatch, useAppSelector } from './state/hooks';
 import JobsMain from './Pages/JobsMain';
 import JobCreation from './Pages/JobCreation';
 
 // eslint-disable-next-line @typescript-eslint/no-empty-interface
 interface Props {}
 
 const App: FC<Props> = () => {
+  const dispatch = useAppDispatch();
+  const getUser = async () => {
+    const user = await axios.get('http://localhost:5000/auth/login/success');
```

Comment:
use relative endpoints for internal API calls

Reply:
just fixed